ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -19,9 +19,9 @@
 
 @dataclasses.dataclass
 class TtsTask:
+    template: str = simple_parsing.field(alias="-T")
     implementation: str = simple_parsing.field(default="azure", alias="-i")
     # Column name containing the text to convert to audio. It can be a Jinja variable expression.
```

Comment:
Move this comment line up before the implementation variable?

Reply:
Actually this comment refers to code that no longer exists, it can be removed.
```suggestion
```